ReadMe.md

```diff
@@ -0,0 +1,12 @@
+## Get Binary Dependencies
+
+In the `build` directory, run
+
+`buildupdate.windows.sh`
```

Comment:
I don't think github readmes should bear the burden of teaching people about git.